projects/plugins/jetpack/modules/carousel/jetpack-carousel.php

```diff
@@ -1035,12 +1132,12 @@ function post_attachment_comment() {
 		do_action( 'jp_carousel_post_attachment_comment' );
 		$comment_status = wp_get_comment_status( $comment_id );
 
-		if ( true == $switched ) {
+		if ( true === $switched ) {
```

Comment:
There's nothing wrong with this, but I wonder why this doesn't just do
```suggestion
		if ( $switched ) {
```
like all the other uses above?

Reply:
Oh good catch, that would make sense.